packages/big-design/src/components/Checkbox/styled.tsx

```diff
@@ -32,25 +32,32 @@ export const StyledCheckbox = styled.label<StyledCheckboxProps>`
     checked || isIndeterminate ? theme.colors.primary : theme.colors.secondary30};
   border-radius: ${({ theme }) => theme.borderRadius.normal};
   color: ${({ theme }) => theme.colors.white};
+  cursor: pointer;
   display: inline-flex;
   height: ${({ theme }) => theme.spacing.large};
   justify-content: center;
-  transition: all 150ms;
+  transition: all 150ms ease-out;
   user-select: none;
   width: ${({ theme }) => theme.spacing.large};
 
+  &:hover {
+    border-color: ${({ checked, isIndeterminate, theme }) =>
+      checked || isIndeterminate ? theme.colors.primary : theme.colors.secondary40};
+  }
+
```

Comment:
This doesn't work well with `disabled` checkboxes. Can you account for this?

Reply:
I have fixed it, however disabled checkboxes seem to be broken at the moment. You can still click on disabled checkboxes.